style.css

```diff
@@ -1046,512 +961,599 @@ border: 1px solid #fff;
+    height: 24px;
+    line-height: 20px;
+    display: flex;
+    align-items: center;
+    left: 25%;
+  }
 
-.frame8 {
-  width: 684px;
-  height: 72px;
-}
+  .languagelist .cssbox {
+    width: 122px;
+    height: 120px;
+  }
 
-.chevron1 {
-  width: 40px;
-  height: 40px;
-  /* right: 26px;
-  top: calc(50% - 40px/2); */
-}
+  .cssimage {
+    width: 48px;
+    height: 48px;
+    position: absolute;
+    top: 69px;
+    border: 1px solid #fff;
+  }
 
-.frameworks {
-  /* left: 24px;
-  right: 78px;
-  top: calc(50% - 24px/2); 
-  position: absolute; */
-  font-size: 20px;
-  line-height: 24px;  
-}
+  .csstext {
+    width: 98px;
+    height: 24px;
+    line-height: 20px;
+    display: flex;
+    align-items: center;
+    left: 46.5%;
+  }
 
-.frame9 {
-  width: 684px;
-  height: 72px;
-}
+  .frame8 {
+    width: 684px;
+    height: 72px;
+  }
 
-.chevron2 {
-  width: 40px;
-  height: 40px;
-  right: 26px;
-  top: calc(50% - 40px/2);
-}
+  .frameworks {
+    position: absolute;
+    height: 24px;
+    left: 24px;
+    font-size: 20px;
+    line-height: 24px;
+  }
 
-.skills {
-  height: 24px;
-  left: 24px;
-  right: 78px;
-  top: calc(50% - 24px/2);
-}
+  .chevron1 {
+    width: 40px;
+    height: 40px;
+    display: flex;
+    flex-direction: row;
+    align-items: flex-end;
+    padding: 8px;
+    position: absolute;
+    left: 75%;
+  }
 
-.seperator1 {
-  width: 684px;
-  position: absolute;
-}
+  .frame9 {
+    width: 684px;
+    height: 72px;
+  }
 
-.seperator2 {
-  width: 684px;
-  position: absolute;
-}
+  .skills {
+    position: absolute;
+    height: 24px;
+    left: 24px;
+    font-size: 20px;
+    line-height: 24px;
+  }
 
-/*Last Page*/
-.contact-form {
-  background-image: url("./sources/contactmedesktop.png");
-  width: 1440px;
-  height: 846px;
-  top: 95%;
-  display: flex;
-  align-items: center;
-  justify-content: center;
-}
+  .chevron2 {
+    width: 40px;
+    height: 40px;
+    display: flex;
+    flex-direction: row;
+    align-items: flex-end;
+    padding: 8px;
+    position: absolute;
+    left: 75%;
+  }
 
-.contactbody {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  padding: 0;
-  gap: 24px;
-  /* position: absolute; */
-  width: 801px;
-  height: auto;
-  /* left: 320px;
-  top: 4421px; */
-}
+  .seperator1 {
+    width: 604px;
+    position: absolute;
+    left: 0;
+    top: 80%;
+  }
 
-.contact-me {
-  width: 801px;
-  height: auto;
-  line-height: 52px;
-  justify-content: center;
-  align-items: center;
-  margin-left: 0;
-}
+  .seperator2 {
+    width: 604px;
+    position: absolute;
+    left: 0;
+    top: 100%;
+  }
 
-.info {
-  width: 569px;
-  height: auto;
-  justify-content: center;
-  text-align: center;
-  margin-left:0;
-}
+  /* Last Page */
+  .contact-form {
+    background-image: url("./sources/contactmedesktop.png");
+    width: 1440px;
+    height: 846px;
+    top: 88%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
```

Comment:
- [x] I would suggest you use a `background image` for the contact section of your project as it is currently missing one, I notice you have the background image in your styling but it's not showing, kindly check the image directory to see if you target it correctly.

- [x] Kindly remove the `border-radius` around the form `input`

Your design 👇 

![joy cont](https://user-images.githubusercontent.com/78634841/208218614-06edeac7-6374-4679-9b9a-d80624640e20.png)


Figma Design 👇 

![cont](https://user-images.githubusercontent.com/78634841/208218639-03c416b7-6dcd-427f-aebf-ae6c7ba5340f.png)